### PR TITLE
fix(eslint): fix remaining eslint issue after config changes

### PR DESCRIPTION
**Pull request by matthieu-crouzet**
Next steps after this PR
- [ ] Rebase with main
- [ ] Migrate new packages (pipeline, clients, training-tools, ...)

### CODE REVIEW

**Review by fpaul-1A — APPROVED** [review]

File: tools/github-actions/new-version/packaged-action/index.js [hunk 1]
```diff
@@ -34055,7 +34055,7 @@ exports.formatGitTagsOutput = formatGitTagsOutput;
  * @param gitOutput response from git command to format
  */
 function formatGitTagsOutput(gitOutput) {
-    return gitOutput.split(/\s+/g)
+    return gitOutput.split(/[\r\n\s]+/g)
```
Comment (kpanot) [inline tools/github-actions/new-version/packaged-action/index.js:34058]: `\s` does not already cover `\n` and `\r`?

File: tools/github-actions/new-version/packaged-action/index.js [hunk 2]
```diff
@@ -34134,23 +34133,22 @@ class NewVersion {
      * @param versionMask
      */
     computeNewVersion(tags, versionMask) {
-        const regexpVersionMask = new RegExp(`^(?:v)?${versionMask}`);
         // Sort tags in descending order and exclude next major in preparation
         let parsedSortedTags = tags
             .map((tag) => semver.parse(tag.replace('V', 'v')))
-            .filter((tag) => !!tag && !!regexpVersionMask.test(tag.raw))
+            .filter((tag) => !!tag && !!tag.raw.match(`^(?:v)?${versionMask}`))
```
Comment (kpanot) [inline tools/github-actions/new-version/packaged-action/index.js:34139]: The previous version was more efficient, why this change?

File: tools/github-actions/new-version/packaged-action/index.js (hunk 1 above)
Comment (matthieu-crouzet) [reply tools/github-actions/new-version/packaged-action/index.js]: issue coming from the build of tools 
it will be fixed after the rebase 
I fallback all changes from tools

File: tools/github-actions/new-version/packaged-action/index.js (hunk 2 above)
Comment (matthieu-crouzet) [reply tools/github-actions/new-version/packaged-action/index.js]: issue coming from the build of tools 
it will be fixed after the rebase 
I fallback all changes from tools

**Review by fpaul-1A — APPROVED** [review]
